client/my-sites/checklist/header.jsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import ScreenReaderText from 'components/screen-reader-text';
+import ProgressBar from 'components/progress-bar';
+
+export class ChecklistHeader extends PureComponent {
+	static propTypes = {
+		total: PropTypes.number.isRequired,
+		completed: PropTypes.number.isRequired,
+		hideCompleted: PropTypes.bool,
+		onClick: PropTypes.func,
+	};
+
+	render() {
+		const { completed, hideCompleted, total, translate } = this.props;
+		const buttonText = hideCompleted
+			? translate( 'Show completed' )
+			: translate( 'Hide completed' );
+
+		/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
Hm,m is this expecting header here instead? Consider renaming this file to `checklist-header.jsx` instead